components/Footer.js

```diff
@@ -5,65 +5,103 @@ export default function Footer() {
 	const year = new Date().getFullYear();
 	return (
 		<div className={styles.Wrapper}>
-			<h2 className={styles.Title}>Erick Askim</h2>
-			<div className={styles.MenuWrapper}>
-				<ul className={styles.Menu}>
-					<li className={styles.Li}>
-						<Link href='/'>
-							<a className={styles.MenuItem}>Home</a>
-						</Link>
-					</li>
-					{/* <li className={styles.Li}>
+			<div className={styles.Container}>
+				<div className={styles.MenuWrap2}>
+					<h2 className={styles.Title}>Erick Askim</h2>
+					<div className={styles.MenuWrapper}>
+						<ul className={styles.Menu}>
+							<li className={styles.Li}>
+								<Link href='/'>
+									<a className={styles.MenuItem}>Home</a>
+								</Link>
+							</li>
+							{/* <li className={styles.Li}>
 						<Link href='/about'>
 							<a className={styles.MenuItem}>About</a>
 						</Link>
 					</li> */}
-					{/* <li className={styles.Li}>
+							{/* <li className={styles.Li}>
 						<Link href='/portfolio'>
 							<a className={styles.MenuItem}>Portfolio</a>
 						</Link>
 					</li> */}
-					<li className={styles.Li}>
-						<Link href='/blog'>
-							<a className={styles.MenuItem}>Blog</a>
+							<li className={styles.Li}>
+								<Link href='/blog'>
+									<a className={styles.MenuItem}>Blog</a>
+								</Link>
+							</li>
+							<li className={styles.Li}>
+								<Link href='/contact'>
+									<a className={styles.MenuItem}>Contact</a>
+								</Link>
+							</li>
+						</ul>
+					</div>
+				</div>
+
+				<div className={styles.FooterWrapper}>
+					<div className={styles.Footer}>
+						<div className={styles.Address}>
+							<h3 className={styles.H3}>Address:</h3>
+							<address className={styles.Link}>
+								536 W La Strada Circle, Medford,
+							</address>
+							<address className={styles.Link}>OR 96504, USA</address>
```

Comment:
That should be one address element. To get the line break use <br /> or fancy css.